lib/fluent/event.rb

```diff
@@ -21,6 +21,10 @@ class EventStream
     include Enumerable
     include MessagePackFactory::Mixin
 
+    def records
```

Comment:
It looks this method returns an integer, not an array. So, both `records` and `events` look bad for me. 
What about `num_events` or just `length`?